packages/chart-advisor/src/auto-chart/config-panel.ts

```diff
@@ -136,6 +142,6 @@ export class ConfigPanel {
     if (type)
       this.iframe.src = `${PATH_PREFIX}/config-panel.html?type=${translate(type)}&cancopydata=${
         this.needCopyData
-      }&lang=${getLanguage()}`;
+      }&lang=${this.language}`;
```

Comment:
@pddpd `autoChart` need set `language` as local option rather than `getLanguage`

Reply:
I see what you mean. In addition, in this case (considering asynchronous loading), do we need to add an optional parameter about default language to `intl.get` method.

Reply:
> I see what you mean. In addition, in this case (considering asynchronous loading), do we need to add an optional parameter about default language to `intl.get` method.

I have committed the above content.